doc/manual.md

```diff
@@ -0,0 +1,676 @@
+concepts written, a user can choose to save the file, if not already
+saved, and then translate these concepts.
+
+![The concepts dialog with a concept file opened.](http://jrbeaumont.github.io/concepts-manual/images/concepts_dialog_Celement.svg)
+
+This is the concepts dialog after
+we have opened the Celement with environment example, named
+“*Celement_with_env_1.hs*”, from the concepts tool examples
+directory. Clicking translate at this point will produce an STG
+representation of these concepts in the workspace.
+
+![The STG produced from translating the concepts.](http://jrbeaumont.github.io/concepts-manual/images/concepts_translated.svg)
+
+The translated concepts will look similar to in the above image.
+
+Now, a user can choose to insert more concepts, make changes to this
+STG, and once they are satisfied with it, can then perform various
+functions on this STG. One can perform transformations, verifications,
+simulations and synthesis on this STG using the menus within this
+workspace now. Any further changes to this STG, based on the results of
+these operations can be made to this STG or to the concepts file.
+
+Importing concepts directly
+---------------------------
+
+In <span><span style="font-variant:small-caps;">Workcraft</span></span>
+it is also possible to import concepts directly from a file, without
+having to view the concepts first. This can be done from the “*File*”
+menu, by selecting the “*Import...*” option.
+
+![The STG produced from translating the concepts.](http://jrbeaumont.github.io/concepts-manual/images/import_menu_screenshot.svg)
+
+When importing concepts using this menu, ensure to set the “*Files of
+Type*” option to “*Concepts file (.hs)*”, as shown in the image above.
+
+Errors
+------
+
+If any errors are encountered during the translation process,
+<span><span style="font-variant:small-caps;">Workcraft</span></span>
+will produce a helpful error message. This usually can tell you with
+more detail what the issue that is causing the error is, but will ask
+you to refer to <span><span
+style="font-variant:small-caps;">Workcraft</span></span>’s console
+window for specific line numbers or signals which need to be corrected.
+These errors will include whether a signal has not been declared as an
+input or output, a signal has not had it’s initial state given, or even
+that the concepts tool has not been installed correctly.
+
+Writing concepts
+=========================================
+
+In this section, we will discuss how to write a concepts file. We will
+use an example and go through it based on lines, explaining what is
+necessary. For this example we will be using the same as example as in
+previous sections, from the examples included with the concepts repo.
+This is “*Celement_with_env_1.hs*”.
+
+Concepts file layout
+--------------------
+
+![A concepts file](http://jrbeaumont.github.io/concepts-manual/images/concepts_file_screenshot.svg)
+
```

Comment:
I suggest to simply use a Haskell code snippet here, e.g.:
```haskell
module Concept where

import Tuura.Concept.Concat
...
```
A screenshop from Sublime Text editor may be confusing (readers might think it's required to use concepts).